pom.xml

```diff
@@ -9,145 +9,86 @@
     <version>1.0-SNAPSHOT</version>
 
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <maven.compiler.source>8</maven.compiler.source>
-        <maven.compiler.target>8</maven.compiler.target>
-
+        <maven.conpiler.version>3.8.1</maven.conpiler.version>
```

Comment:
Hi anh Khiem, please correct the gramma in the "maven.conpiler.version", should change it to "maven.compiler.version"